Sources/SnapshottingTests/PreviewTest.swift

```diff
@@ -6,166 +6,49 @@
 //
 
 import Foundation
+import SnapshotPreviewsCore
 import SnapshottingTestsObjc
-import MachO
+import SwiftUI
 
-open class PreviewTest: EMGPreviewBaseTest {
+// Test Xcode previews by forcing a layout pass of each one
+open class PreviewTest: EMGPreviewBaseTest, PreviewFilters {
```

Comment:
`PreviewTest` naming is a bit vague IMO, I could see someone confused with this and `SnapshotTest`. Maybe `HasNoCrashingPreviewsTest` since that's the intent? I guess it could also discover 0x0 sized snapshots too?

Reply:
I find `PreviewLayoutTest` (what noah picked) to still be sort of vague. @noahsmartin other than layout crash detection, is there any other use for this you could think of?

If not, I'd suggest we make that more clear and put that in the name ex. `PreviewLayoutCrashTest` or `NoCrashingPreviewLayoutsTest`